lib/src/models/card.dart

```diff
@@ -38,10 +38,10 @@ class Card implements Comparable {
   /// Creates a [Card] from Rank and Suit.
   ///
   /// ```dart
-  /// final aceOfSpade = Card(Rank.ace, Suit.spade);
-  /// final deuceOfHeart = Card(Rank.deuce, Suit.heart);
-  /// final TreyOfDiamond = Card(Rank.trey, Suit.diamond);
-  /// final kingOfClub = Card(Rank.king, Suit.club);
```

Comment:
Creating Card with `Card(Rank.ace, Suit.spade)` is not possible now as we have to pass in the arguments `rank` and `suit`